### ISSUE TITLE
add arrow keys for next and previous track

### ISSUE THREAD

**Issue by ghondar** [andrewngu/sound-redux#29]
I feel that it is more intuitive to use the right and left arrows to change the music, are mostly used in the majority of players.

I was watching that using keypress and therefore cannot capture the arrows key events. I see that you use keypress instead of keydown therefore can not capture the events of the arrows.

Another solution would be to use [mousetrap](https://craig.is/killing/mice) for keyboard events, being a way to capture more easy to maintain in the development, just add 2kbs to the project

**Comment by andrewngu** [comment]
Makes sense :+1: Although I would prefer just using keydown instead of bringing in another library.

**Comment by ghondar** [comment]
OK, I already have a pr by adding arrows events using keydown, also i added notifications to deezer style, in case you're in another tab.
![Notification]
(http://i.imgur.com/1j5vlMA.png)

**Comment by andrewngu** [comment]
Cool, but I'd prefer to have the notifications be a setting that users opt into. It'd be annoying if I suddenly started getting notifications without an explanation. What do you think?

**Comment by ghondar** [comment]
of course, the first time load you page, ask if you want to receive notifications

**Comment by ghondar** [comment]
Google will manage notifications

![Notification Manager]
(http://i.imgur.com/jyNPJnR.png)

**Comment by andrewngu** [comment]
Sounds good. I just remember turning on notifications for a website one time, not knowing exactly what would happen (new news articles) and got super annoyed. Is it possible to describe exactly what the notifications will do when asking for permission?

**Comment by ghondar** [comment]
I think that it's not possible,When you run Notification.requestPermission, it only shows a popup saying: "https://soundredux.io" wants to show notifications -> Block or Allow. 
[requestPermission](https://developer.mozilla.org/en-US/docs/Web/API/Notification/requestPermission)
![request]
(http://i.imgur.com/d5dSi3V.png)

**Comment by wub** [comment]
What a few sites do, when they want to enable notifications, is have an "Enable notifications" checkbox/button underneath a paragraph or two explaining exactly what and why, in some kind of settings area.

It gives them an easy way to turn them on and off, and lets them know what it'll do.

**Comment by wub** [comment]
Also, for those who don't know - <kbd>shift</kbd> + <kbd>&larr;</kbd> / <kbd>shift</kbd> + <kbd>&rarr;</kbd> will go back and forth between songs, just like SoundCloud!

**Comment by wub** [comment]
@andrewngu Should this be closed now?
